gateway/src/apicast/policy/scope_check/scope_check.lua

```diff
@@ -0,0 +1,105 @@
+local policy = require('apicast.policy')
+local _M = policy.new('Scope Check Policy')
+
+local ipairs = ipairs
+local MappingRule = require('apicast.mapping_rule')
+
+local new = _M.new
+
+local function init_mapping_rules(scopes)
+  for _, scope in ipairs(scopes) do
+    scope.mapping_rule = MappingRule.from_proxy_rule({
+      http_method = 'GET',
+      pattern = scope.resource,
+      querystring_parameters = {},
+      metric_system_name = 'hits',
```

Comment:
I know why this is here: https://github.com/3scale/apicast/blob/db9f97e430d1ed01abf78b7aeddfbc493a4a2fc8/gateway/src/apicast/mapping_rule.lua#L89

This is task for us: improve MappingRule be allowed initialize without metric system name, so it can be used for such things. 

`delta` should not be needed too (because we just do matching, not incrementing metrics).

Reply:
Removed `delta`.